src/test/java/domain/RoundResultTest.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+import movingStrategy.AlwaysMove;
+import movingStrategy.NeverMove;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class RoundResultTest {
+
+    @DisplayName("1회 진행 시 결과를 저장")
+    @Test
+    void saveProgress() {
+        RacingCar car1 = new RacingCar(new AlwaysMove());
+        RacingCar car2 = new RacingCar(new NeverMove());
```

Comment:
```suggestion
        RacingCar car1 = new RacingCar(new AlwaysMove());
        RacingCar car2 = new RacingCar(new NeverMove());
```
`AlwaysMove`, `NeverMove` 전략패턴을 `RacingCar`에게 가지게할필요가 있을까요? 
`RacingCar`를 관리하는 `RacingGame`에서 가지면 어떨까요?

Reply:
꼭 `move`를 통해서만 `position`을 변경 가능할까요?
생성자를 다양하게 만들수있지 않을까요?

```
int position = 1;
RacingCar car = RacingCar(position)
```

Reply:
위에서 언급된 `lastCarPosition`과 같은 맥락에서 질문이 있는데요.
`lastCarPosion`이나 `position`을 변수로 받는 `RacingCar` 생성자의 경우
테스트 코드에서만 사용되는 메서드인데 main 패키지에 이러한 코드를 노출해도 되는지 의문이 있었습니다.

사실 `AlwaysMove`나 `NeverMove`도 테스트 코드에서만 사용되는 객체인 건 마찬가지인데요.
이러한 경우에 어떻게 처리하는 게 좋을지 의견 주시면 감사하겠습니다!